### PR TITLE
add fixed machine-translation-nar-{ru-en} models

### PR DESCRIPTION
**Pull request by bes-dev**
Add updated machine-translation-nar-{ru-en}-0002 models (CVS-76517)

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by vladimir-dudnik** [comment]
@bes-dev please add model reference to [device_support.md](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/device_support.md) file

**Comment by bes-dev** [comment]
> @bes-dev please add model reference to [device_support.md](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/device_support.md) file

Done

**Review by eaidova — APPROVED** [review]

File: models/intel/machine-translation-nar-en-ru-0002/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| MParams           | 69.29                 |
+| Source framework  | PyTorch\*             |
+
+## Accuracy
+
+The quality metrics were calculated on the wmt19-ru-en dataset ("test" split in lower case).
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| BLEU                      |        21.6 % |
```
Comment (Anna-Mironova) [inline models/intel/machine-translation-nar-en-ru-0002/README.md:23]: update accuracy value in README files and reference in AC configs, please

File: models/intel/device_support.md [hunk 2]
```diff
@@ -60,6 +60,8 @@
 | machine-translation-nar-en-de-0002 | YES | YES |    |
 | machine-translation-nar-en-ru-0001 | YES | YES |    |
 | machine-translation-nar-ru-en-0001 | YES | YES |    |
+| machine-translation-nar-en-ru-0002 | YES | YES |    |
```
Comment (eaidova) [inline models/intel/device_support.md:63]: probably if model gives on gpu 0.0 bleu, it can not be called supported for device?

Comment (vladimir-dudnik) [reply models/intel/device_support.md:63]: definetely, @bes-dev please leave only CPU as supported device

Comment (bes-dev) [reply models/intel/device_support.md]: Agree with you. Removed GPU support in device_support.md for all machine-translation-nar-* models.
Done

**Review by vladimir-dudnik — APPROVED** [review]

File: models/intel/machine-translation-nar-en-ru-0002/README.md (hunk 1 above)
Comment (bes-dev) [reply models/intel/machine-translation-nar-en-ru-0002/README.md]: Done

File: models/intel/machine-translation-nar-en-ru-0002/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# machine-translation-nar-en-ru-0002
+
+## Use Case and High-Level Description
+
+This is a English-Russian machine translation model based on non-autoregressive Transformer topology.
+
+Tokenization occurs using the SentencePieceBPETokenizer (see the demo code for implementation details) and the enclosed tokenizer_src and tokenizer_tgt folders.
```
Comment (vladimir-dudnik) [inline models/intel/machine-translation-nar-en-ru-0002/README.md:7]:
```suggestion
Tokenization occurs using the SentencePieceBPETokenizer (see the demo code for implementation details) and is enclosed in tokenizer_src and tokenizer_tgt folders.
```

**Pull request merged**